### PR TITLE
Adds Tychus Glasses, Limb Autosurgeons, and the Jensen Syndikit Special Bundle

### PR DESCRIPTION
**Pull request by Skrem7**
# Document the changes in your pull request

Adds tychus glasses, an item meant to use the unused sprites that we have for Adam Jensen glasses. There's a dummy variant in the theater drobe which can extend and retract, but doesn't actually do anything beyond look "cool". Adds an agent version which, when extended, provides thermals and night vision, but has flash_protect -2. This agent version is only obtainable through the bundle included in this PR

Also adds limb autosurgeons, which are autosurgeons specifically designed to take limbs (primarily cyborg limbs) for augmentation that does not require a helping hand. At the moment, the only way to access five of the six preset autosurgeons is through the Jensen Syndikit Special bundle

The Jensen Syndkit Special bundle of weight 2 is themed around the character of the same name from Deus Ex: Human Revolution and Deus Ex: Mankind Divided. The kit (roughly cost 36 TC) is as follows:
- Agent Tychus Glasses
- Tactical Turtleneck
- Jump Boots
- Syndicate Encryption Key
- Syndicate Toolbox
- A paper guide on the order in which to use the autosurgeons and implants that come with the kit
- Cyborg Chest Limb Autosurgeon
- Cyborg Right Arm Limb Autosurgeon
- Cyborg Left Arm Limb Autosurgeon
- Cyborg Right Leg Limb Autosurgeon
- Cyborg Left Leg Limb Autosurgeon
- Welding Eyes Autosurgeon (newly added subtype included in this PR)
- Two mantis blade autosurgeons
- Adrenaline implant

Also glances some existing autosurgeons with uses so they can't be exploited because having an autosurgeon with infinite uses is REALLY really powerful and generally an oversight of these specific ones.

# Spriting
Image of the icon state of the shades with the lens retracted:
image

Old one with the lens extended:
image

Mob sprites are almost impossible to see so here's the naked improved old extended:
image

Retracted:
image

Sunglasses for reference:
image

# Wiki Documentation

Bundle to be added to Syndicate items under Syndikit Special
Adds the tychus glasses to the theater vendor
Adds the agent tychus glasses to the eyewear section in Clothing & Accessories

# Changelog

:cl:  
rscadd: Adds a welding eyes autosurgeon
rscadd: Adds a new type of autosurgeon, the limb autosurgeon, and creates six types of it; cyborg chest, head, arms, and legs
rscadd: Adds the tychus glasses, which are based of off Adam Jensen's iconic eyewear from Deus Ex: HR and MD. They can be bought from the theater drobe, but give no special benefits
rscadd: Adds the Jensen Syndikit Special, a kit themed around augmentation with vague implementations of a small portion of Jensen's kit
rscadd: Adds the agent tychus glasses for the Syndikit, which, while their lens are extended, provide thermals and night vision at the cost of extreme flash vulnerability
imageadd: Technically slightly tweaks the existing jensen glasses sprite and adds one for the lens being lowered
/:cl:

### CODE REVIEW

File: code/modules/clothing/glasses/_glasses.dm [hunk 1]
```diff
@@ -289,6 +289,27 @@
...
+/obj/item/clothing/glasses/welding/jensen
+	name = "tychus glasses"
+	desc = "A set of technical shades that link with the user when worn, allowing them to retract and extend the lens at will."
+	icon_state = "jensenshades"
+	item_state = "jensenshades"
+	flash_protect = 0
+	tint = 0
+
+/obj/item/clothing/glasses/welding/jensen/agent
+	desc = "A modified set of technical shades that can retract and extend at will. The lens provide night vision and thermal sight but are highly vulnerable to bright lights."
```
Comment (VaelophisNyx) [inline code/modules/clothing/glasses/_glasses.dm]: "the lens**es** provide night vision"

File: code/modules/vending/autodrobe.dm [hunk 2]
```diff
@@ -225,7 +225,8 @@
 		           /obj/item/clothing/under/yogs/rycliesuni = 1,
 		           /obj/item/clothing/head/yogs/toad = 1,
 		           /obj/item/clothing/head/helmet/justice = 1,
-		           /obj/item/clothing/mask/yogs/richard = 1) //yogs end
+		           /obj/item/clothing/mask/yogs/richard = 1,
+				   /obj/item/clothing/glasses/welding/jensen = 1) //yogs end
```
Comment (VaelophisNyx) [inline code/modules/vending/autodrobe.dm:229]: Quirked up autodrobe stock?

Comment (Skrem7) [reply code/modules/vending/autodrobe.dm:229]: ?

Comment (VaelophisNyx) [reply code/modules/vending/autodrobe.dm:229]: dont these provide thermals, or was that something else

Comment (Skrem7) [reply code/modules/vending/autodrobe.dm:229]: different item

**Comment by CMOisLing** [comment]
"I never asked for this"

**Comment by RG4ORDR** [comment]
Why contain it, s'cool.

**Comment by ilikesaying** [comment]
"Why are you locked in the bathroom?"

**Review by KillerOrcaCora — APPROVED** [review]

**Comment by JamieD1** [comment]
Do you not think this is alot for one bundle?

**Comment by Skrem7** [comment]
> Do you not think this is alot for one bundle?

Could lose adrenaline implant and be fine probably. Only reason I tweaked it up is because one gains a pretty hefty weakness to EMPs using this.

**Comment by Skrem7** [comment]
Gonna split up sections of this into different PRs

**Pull request closed without merging**